frontend/package.json

```diff
@@ -109,17 +109,20 @@
     "prettier": "^3.0.3",
     "prettier-plugin-tailwindcss": "^0.5.4",
     "sass": "^1.67.0",
-    "stylelint": "^14.16.1",
+    "stylelint": "^15.10.0",
     "stylelint-config-idiomatic-order": "^9.0.0",
-    "stylelint-config-prettier-scss": "^0.0.1",
+    "stylelint-config-prettier-scss": "^1.0.0",
     "stylelint-config-recommended-vue": "^1.5.0",
-    "stylelint-config-standard-scss": "^6.1.0",
+    "stylelint-config-standard-scss": "^10.0.0",
     "tailwindcss": "^3.3.3",
     "typescript": "^5.2.2",
     "vite": "^4.4.9",
     "vite-plugin-environment": "^1.1.3",
     "vite-plugin-pages": "^0.31.0",
     "vitest": "^0.34.4",
     "vue-tsc": "^1.8.11"
+  },
+  "overrides": {
```

Comment:
Until https://github.com/ream88/stylelint-config-idiomatic-order/issues/78 is solved